source/index.html.md

```diff
@@ -639,10 +569,102 @@ Name | Type | Description
         on_close: () => {
           console.log('on_close!');
         }
-      });
-      highlineLink.open();
-    }
+    });
+    highlineLink.open();
   </script>
 </body>
 </html>
+```
+
+
+## Building the payload
+### One-Step Payment Connection
+```javascript
+POST highline_link/encode
+Basic Auth
+Request:
+{
+ "enrollment_id",
+ "feature": "direct_deposit_payment"
+}
+ 
+Response:
+{
+ "payload": "encrypted_value"
+}
+
+```
+
+### Two-Step Payment Connection
```

Comment:
Direct Deposit Switch.

And then put all the steps in this section including encode/decode and encrypted payload endpoints